js-exercises/last-index-of/lastIndexOf.js

```diff
@@ -0,0 +1,5 @@
+
+const lastIndexOf = (item, itemArray) => itemArray.lastIndexOf(item)
```

Comment:
I think you may have written your own function, instead of using built-in indexOf.